src/modules/hip/kernel/jitter.hpp

```diff
@@ -0,0 +1,315 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+#include "rng_seed_stream.hpp"
+
+__device__ void jitter_roi_and_srclocs_hip_compute(int4 *srcRoiPtr_i4, RpptXorwowStateBoxMuller *xorwowState, uint kernelSize, uint bound, int id_x, int id_y, d_float16 *locSrc_f16)
+{
+    d_float8 nhx_f8, nhy_f8;
```

Comment:
nhx_f8, nhy_f8  does not sound intuitive to me
Please check and give better names for this variables

Reply:
Done